README.md

```diff
@@ -428,6 +428,53 @@ Example
 
 ## Methods
 
+- **invalidate**
+
+### `cache.invalidate(references, [storage])`
+
+`cache.invalidate` perform invalidation over the whole storage; if `storage` is not specified, using the same `name` as the defined function, invalidation is made over the default storage.  
```

Comment:
I don't think this is particularly clear. Looking at the examples, it seems like the `storage` option needs to be the name of a policy e.g. `'Query.countries`'. Is that correct?